frontend/src/scenes/actions/Action.tsx

```diff
@@ -41,6 +43,10 @@ export function Action({ id }: { id?: ActionType['id'] } = {}): JSX.Element {
         return <NotFound object="action" />
     }
 
+    if (getAppContext()?.resource_access_control?.[AccessControlResourceType.Action] !== AccessControlLevel.Editor) {
```

Comment:
Functionally this all seems fine to me.

Thinking more product-wise though, actions are used in so many other places like insights, the list view for filtering events etc. Is this going to cause issues there if someone has no access 🤔 

If it doesn't cause issues is it going to be weird that you can see them elsewhere in the UI just not here? 

Feels like this might be a case of a kind of sub-resource where "no access" actually doesn't make a lot of sense. I dunno

Reply:
to be clear i think the ability to restrict editing makes total sense, just restricting viewing feels like it might have external consequences

Reply:
that's a good point. I'll explore this more. I think every resource it going to have some customizations so need to figure out how to organize these.